dataflow/manual/content.tex

```diff
@@ -1083,6 +1078,23 @@ \subsubsection{Assert statements}
 is no alternate control flow?
 \end{workinprogress}
 
+\subsubsection{Default case and fall through for switch statement}
+Switch statement is handled as a chain of \code{CaseNode} and nodes for
+the case. \code{CaseNode} makes a branch by comparing the equality of
+the expression of the switch statement and the expression of the case.
+Note that the expression of switch statement must be executed just only
+once at first of switch statement. To refer the value of the expression,
+a fake variable is created and the expression is assigned to a fake
+variable. \code{THEN_TO_BOTH} edge goes to nodes for the case and \code{ELSE_TO_BOTH}
+edge goes to next \code{CaseNode}. When a next is default case, it goes
+to nodes for default case. If a break statement in nodes for a case create
+an edge to next node of the switch statement. When it doesn't exist, it means
+that it's fall-through and the next node of nodes of the case is nodes of
+the next case. For example, the code in \autoref{lst:CFGSwitch} produces
+the control flow graph in \autoref{fig:CFGSwitch}. The face variable named
```

Comment:
Should the `face` be `fake`? Or something else?
Can you do another pass through the English of this paragraph?